src/rai_toolbox/datasets/cifar_corruptions.py

```diff
@@ -189,7 +189,7 @@ def __init__(
         if not isinstance(severity, (int, Integral)) and 1 <= severity <= 5:
             raise ValueError(f"`severity` must be an integer in [1, 5]. Got {severity}")
 
-        severity -= 1  # type: ignore
+        severity -= 1  # ignore
```

Comment:
`# ignore` doesn't mean anything. `# type: ignore` tells the type-checker that it should ignore some error. What is the reasoning for this diff?

Reply:
this was an errant auto-formatting fix from one of the pre-commit tests. fixed it with the commit below
![image](https://user-images.githubusercontent.com/33703298/182655405-0f34bc9c-8741-4f9b-a0c0-959bf84afe39.png)